src/pynwb/icephys.py

```diff
@@ -116,10 +116,7 @@ class CurrentClampSeries(PatchClampSeries):
              'default': 'volts'})
     def __init__(self, **kwargs):
         name, data, unit, electrode, gain = popargs('name', 'data', 'unit', 'electrode', 'gain', kwargs)
-        if unit != 'volts':
-            warnings.warn("Unit for %s '%s' is ignored and will be set to 'volts' as per NWB 2.1.0."
-                          % (self.__class__.__name__, name))
-        unit = 'volts'
+        unit = ensure_unit(self, name, unit, 'volts', '2.1.0')
```

Comment:
BTW ignorant question:  It is `CurrentClampSeries`.  Current is measured in `amperes` -- why is it `volts` here?  There is similarly opposite below for `VoltageClampSeries`. 
@rly?

Reply:
@yarikoptic - a current clamp measures voltage and voltage clamp measures current. https://en.wikipedia.org/wiki/Electrophysiology#Intracellular_recording

Reply:
gotcha -- thank you @satra!